### PR TITLE
Fix: explicitly enforce email to be the first value of csv file

### PR DESCRIPTION
**Pull request by Zer0Legion**

### CODE REVIEW

**Comment by Zer0Legion** [comment]
Codecov is failing me to a dumb reason, but I have already tested all 3 conditions of the line it is making noise about:
```
boolean isEmailFirstValue(List<String[]> lst) {
        return lst.size() > 0 && lst.get(0).length > 0 && lst.get(0)[0].equals(HEADER_EMAIL);
    }

**Comment by delishad21** [comment]
Codecov is throwing a tantrum likely because it wants all possible permutations of the three conditions in that statement 
2^3 permutations -> {TTT FFF TFF TFT FTT FTF FFT TTF}. Might be easier to test if you break it up

**Review by delishad21 — COMMENTED** [review]
This stops our export import function from being circularly compatible but I think its still a good idea to split the import and importExamScores function to minimise bugs

File: src/main/java/seedu/address/logic/commands/ImportExamScoresCommand.java [hunk 1]
```diff
@@ -191,16 +195,28 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
         List<String[]> lst = CsvUtil.readAllLinesForImportExamScores(filepath);
-        reverse(lst);
-        HashMap<String, HashMap<String, Double>> headers = createExamsMapping(lst);
+        System.out.println(lst);
+        if (!isEmailFirstValue(lst)) {
```
Comment (jayllo-c) [inline src/main/java/seedu/address/logic/commands/ImportExamScoresCommand.java:199]: great job abstracting the checks here

File: src/main/java/seedu/address/logic/commands/ImportExamScoresCommand.java [hunk 2]
```diff
@@ -271,4 +287,8 @@ private static void reverseElements(String[] row, String[] reversedRow) {
         }
     }
 
+    boolean isEmailFirstValue(List<String[]> lst) {
```
Comment (jayllo-c) [inline src/main/java/seedu/address/logic/commands/ImportExamScoresCommand.java:290]: functionality well coded

**Review by jayllo-c — COMMENTED** [review]
Well implemented code that will definitely minimize potential feature flaws

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 3]
```diff
@@ -20,7 +20,7 @@ public class DeleteCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Parameter: INDEX (must be a positive integer)\n"
```
Comment (danielleloh) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:23]: good catch for all the error message formatting

**Pull request merged**